docs/fragments/pr-sanitizer.adoc

```diff
@@ -0,0 +1,51 @@
+== PR Sanitizer Plugin
+
+This plugin enforces that the PR title comforms with the semantic commit message style. Conformance with the semantic commit message style not only makes your changelog and git history clean and easy to navigate but also encourages making atomic commits that are more clearly understood.
```

Comment:
How about adding  `inspired by` https://seesparkbox.com/foundry/semantic_commit_messages ?

Reply:
Not sure if want to link to this? WDYT @MatousJobanek?